api/server/util.go

```diff
@@ -0,0 +1,28 @@
+package server
```

Comment:
You can remove this entire file. No need to support ownership in the old REST api.